chaos_genius/alerts/slack.py

```diff
@@ -173,6 +154,7 @@ def _format_slack_anomalies(
     top10: "Sequence[anomaly_alerts.AnomalyPointFormatted]",
     kpi_name: Optional[str] = None,
     include_kpi_link: bool = True,
+    include_kpi_name: bool = True,
```

Comment:
@Amatullah looks like both of them are True/False at the same time (they are always the same value) and I think `include_kpi_link` was used for the same purpose that `include_kpi_name` is being used for now. So I think one of them is redundant.

Reply:
Fixed.